src/popups/TarifverbundkartePopup/TarifverbundkartePopup.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Feature from 'ol/Feature';
+import { useTranslation } from 'react-i18next';
+import Link from '../../components/Link';
+import './TarifverbundkartePopup.scss';
```

Comment:
Use material ui styles. we try to avoid scss file now

Reply:
Removed scss